tagilmo/utils/vereya_wrapper.py

```diff
@@ -436,11 +440,22 @@ def observeProcCached(self):
     def changed(self, name):
         pass
 
+    def cleanRecipes(self, recipes):
+        if recipes is not None:
+            for i in range(len(recipes)):
+                recipes[i]['name'] = recipes[i]['name'].replace("block.minecraft.", "").replace("item.minecraft.", "")
+                recipes[i]['ingredients'] = list(filter(None, recipes[i]['ingredients']))
+                for j in range(len(recipes[i]['ingredients'])):
+                    if len(recipes[i]['ingredients'][j][0]) <= 0:
+                        continue
+                    recipes[i]['ingredients'][j][0]['type'] = recipes[i]['ingredients'][j][0]['type'].replace("block.minecraft.", "").replace("item.minecraft.", "")
```

Comment:
Why only one alternative of ingredients is considered?
Also, I'm not sure if it is really better to explicitly take the structure of receipts into account, or to use a generic function like this:
```
    def remove_mcprefix_rec(data):
        if isinstance(data, str):
            return data.split('.')[-1] if 'minecraft' in data else data
        if isinstance(data, dict):
            r = {}
            for k in data.keys():
                r[remove_mcprefix_rec(k)] = remove_mcprefix_rec(data[k])
            return r
        if isinstance(data, list):
            return remove_mcprefix_rec(l) for l in data]
        return data
```

Reply:
@Necr0x0Der works like a charm. Thanks. I've added to PR